python/lsst/ts/electrometer/controller.py

```diff
@@ -220,6 +221,13 @@ async def connect(self) -> None:
                 f"Expected:\n {expected} \n but got: \n{res} \n"
             )
             raise RuntimeError("Communication verification failed.")
+        if self.reset:
```

Comment:
Is there a reason not to run this every time we connect to it? 
The CSC won't know whether or not you have power cycled the unit while the CSC was in standby, so I think you have to assume that it was and just run this all the time.

Reply:
As it's coded right now, when you transition to enabled, to standby, then come back to enabled, this piece of code wouldn't get run.

Reply:
Good point. It should be fine to run every time. I should test it first though.

Reply:
Yes, that should be fine. I've removed the `self.reset` variable completely.